src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -166,6 +166,7 @@ internal static class IDEDiagnosticIds
         public const string SimplifyPropertyPatternDiagnosticId = "IDE0170";
 
         public const string UseTupleSwapDiagnosticId = "IDE0180";
+        public const string UseParameterNullCheckingId = "IDE0181";
```

Comment:
190.  we are trying to give some room to items so taht if we add more cases for them their IDs can be grouped nicely.  that's why we have the jumps from 100/110/120/130/etc.  thanks!

Reply:
Have made the change.

Reply:
@CyrusNajmabadi I'm a bit confused. It seems like we're always making jumps regardless and are missing the purpose of jumps in the first place. To be specific, why not group this new diagnostic with `UseNullCheckOverTypeCheckDiagnosticId`? Anything I'm missing about the idea of "grouping"?